ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -234,6 +236,10 @@ public static KsqlRestConfig convertToApiServerConfig(final KsqlRestConfig confi
           origs.put(ApiServerConfig.TLS_CLIENT_AUTH_REQUIRED, "required");
         }
       }
+      final List<String> unauthedPaths = config.getList(RestConfig.AUTHENTICATION_SKIP_PATHS);
+      if (unauthedPaths != null) {
+        origs.put(ApiServerConfig.AUTHENTICATION_SKIP_PATHS_CONFIG, unauthedPaths);
+      }
```

Comment:
Seems this is set only if `keyStoreLocation` is not null/empty. Why is that?

Reply:
Doh!